dev/NumberBox/TestUI/NumberBoxPage.xaml

```diff
@@ -0,0 +1,148 @@
+                Header="MinMaxMode"
+                      SelectedValue="None"
+                      SelectionChanged="Bounds_Changed">
+                    <x:String>None</x:String>
+                    <x:String>MinEnabled</x:String>
+                    <x:String>MaxEnabled</x:String>
+                    <x:String>MinAndMaxEnabled</x:String>
+                    <x:String>WrapEnabled</x:String>
+                </ComboBox>
+
+
+
+
+
+                <TextBlock Text="Min" RelativePanel.RightOf="min" RelativePanel.Below="boundbox" />
+                <controls:NumberBox x:Name="min"
+                RelativePanel.Below="boundbox"/>
+                <TextBlock Text="Max" RelativePanel.RightOf="max" RelativePanel.Below="min" />
+
+                <controls:NumberBox x:Name="max"
+                RelativePanel.Below="min"/>
+
+                <TextBlock Text="Fraction Digits" RelativePanel.RightOf="frac" RelativePanel.Below="max" />
+                <controls:NumberBox x:Name="frac" RelativePanel.Below="max" Value="0"/>
+
+                <TextBlock Text="Integer Digits" RelativePanel.RightOf="intdig" RelativePanel.Below="frac" />
+                <controls:NumberBox x:Name="intdig" RelativePanel.Below="frac" Value="1" />
+
+                <TextBlock Text="Significant Digits" RelativePanel.RightOf="sigdig" RelativePanel.Below="intdig" />
+                <controls:NumberBox x:Name="sigdig" RelativePanel.Below="intdig" />
+
+                <CheckBox x:Name="dec" RelativePanel.Below="sigdig" Content="IsDecimalPointAlwaysDisplayed" />
+                <CheckBox x:Name="zerosign" RelativePanel.Below="dec" Content="IsZeroSigned" />
+
+                <ComboBox Header="Number Rounder Type (Increment/SigFig)" x:Name="rounder" PlaceholderText="NumberRounder" RelativePanel.Below="zerosign"  SelectedValue="IncrementNumberRounder" >
+                    <x:String>IncrementNumberRounder</x:String>
+                    <x:String>SignificantDigitsNumberRounder</x:String>
+                    <x:String>None</x:String>
+
+                </ComboBox>
+
+
+                <ComboBox x:Name="algo" Header="Rounding Algorithm" RelativePanel.Below="rounder" SelectedValue="RoundAwayFromZero" >
+                    <x:String>None</x:String>
+                    <x:String>RoundAwayFromZero</x:String>
+                    <x:String>RoundDown</x:String>
+                    <x:String>RoundHalfAwayFromZero</x:String>
+                    <x:String>RoundHalfDown</x:String>
+                    <x:String>RoundHalfToEven</x:String>
+                    <x:String>RoundHalfToOdd</x:String>
+                    <x:String>RoundHalfTowardsZero</x:String>
+                    <x:String>RoundHalfUp</x:String>
+                    <x:String>RoundTowardsZero</x:String>
+                    <x:String>RoundUp</x:String>
+                </ComboBox>
+
+
+
+                <TextBlock Text="IncrementPrecision (For inc rounder)" RelativePanel.RightOf="algo" RelativePanel.Below="algo" />
```

Comment:
this should be `RelativePanel.RightOf="increment"`
Making this change will mean that the label appears next to the item it is labeling rather than having a big gap between the two.